pkgs/development/python-modules/lesscpy/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+
+, ply
+, six
+
+, tox
+, nose
+, flake8
+}:
+
+
+buildPythonPackage rec {
+  pname = "lesscpy";
+  version = "0.15.1";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-EEXRepj2iGRsp1jf8lTm6cA3RWSOBRoIGwOVw7d8gkw=";
+  };
+
+  doCheck = true;
+
+  checkInputs = [
+    tox
+    nose
+    flake8
+  ];
+
+  propagatedBuildInputs = [
+    six
+    ply
+  ];
+
```

```suggestion

  pythonImportsCheck = [ "lesscpy" ];

```

Maybe add a `pythonImportsCheck` here?